UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/IEventHubLoggingHealthClient.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    public interface IEventHubLoggingHealthClient
+    {
+        public Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default);
```

Comment:
Access specifier not needed in an interface, please remove:
```suggestion
        Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default);
```

Reply:
Access specifier removed